beginner_source/nn_tutorial.py

```diff
@@ -75,6 +75,12 @@
 import numpy as np
 
 pyplot.imshow(x_train[0].reshape((28, 28)), cmap="gray")
+# pyplot.show() only if not on colab
+try:
+    import google.colab
+except ImportError:
+    pyplot.show()
+    pass
```

Comment:
Why is `pass` needed?

Reply:
pass is not needed given pyplot.show(). Sorry!